iceberg/openhouse/internalcatalog/src/main/java/com/linkedin/openhouse/internal/catalog/repository/HouseTableRepositoryImpl.java

```diff
@@ -159,14 +159,14 @@ private Mono<? extends HouseTable> handleHtsHttpError(Throwable e) {
     } else if (e instanceof WebClientResponseException
         && ((WebClientResponseException) e).getStatusCode().is5xxServerError()) {
       return Mono.error(
-          new HouseTableRepositoryStateUnkownException(
+          new HouseTableRepositoryStateUnknownException(
               "Cannot determine if HTS has persisted the proposed change", e));
     } else if (ExceptionUtils.indexOfThrowable(e, DnsNameResolverTimeoutException.class)
         != -1) { // DnsNameResolverTimeoutException appears nested within exception causes and
       // ExceptionUtils class is used to match the occurrence of this failure. Retry is done
       // for this failure using existing retry template.
       return Mono.error(
-          new HouseTableRepositoryStateUnkownException(
+          new HouseTableRepositoryStateUnknownException(
               "HTS service could not be resolved due to DNS lookup timeout", e));
     } else {
       return Mono.error(new RuntimeException("UNKNOWN and unhandled failure from HTS:", e));
```

Comment:
Should we do a pass on this method to make sure any network related exception will not get us into a similar situtation?

Reply:
Added a test